CMakeLists.txt

```diff
@@ -80,6 +80,14 @@ add_executable(
     src/tn93_shared.cc
 )
 
+add_executable(
+    fasta_digest
+    src/fasta_digest.cpp
```

Comment:
This file is missing. @spond -- Would it be possible to add fasta_digest to the repo?

Reply:
Thank you for noticing. Done.